### DISCUSSION TITLE
lдай свои контакты.

### DISCUSSION THREAD

**Discussion by iPowerLife** [hldh214/lok_bot#5] [Show and tell]
как с тобой связаться? телеграмм или дискорд

**Comment by hldh214** [comment]
Currently we don't have telegram or discord group, I think discussions here is enough.

**Reply by iPowerLife** [reply]
ну я для личных целей) а так для будущего, готов тестировать данного бота и хочется чтобы он был популярным и платным для вас. чтобы вы получали деньги на кофе)
